kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -657,13 +631,9 @@ extension [A, S](effect: A < (S & Choice))
       * @return
       *   A computation that produces the result of this computation with Abort[E] effect
       */
-    def choiceToAbort[E, S1](error: => E < S1)(
-        using
-        Flat[A],
-        Frame
-    ): A < (S & S1 & Abort[E]) =
+    def choiceToAbort[E](error: E)(using Flat[A], Frame): A < (S & Abort[E]) =
```

Comment:
error should be by name

pre-existing: combinator should return `Seq` or mention it's taking `head`

Reply:
What do you think about removing these methods instead? It seems a foot gun even if we make scaladocs and naming more clear

Reply:
removed